### ISSUE TITLE
Conda installation for pandastable latest version

### ISSUE THREAD

**Issue by parvathycec** [dmnfarrell/pandastable#75]
Hi, 
I wanted to install pandastable latest version to conda. Current conda install is having conflicts with many other libraries. 
However, I did pip install in the conda python and got it working for now.
Thanks

**Comment by dmnfarrell** [comment]
Yes the conda version is out of date. pip will always be the latest. I'll leave this open until I go back and try build the conda package again.
